HACKING.md

```diff
@@ -115,6 +118,18 @@ import (
 )
 ```
 
+### Log and error messages
+
+**Log messages** (i.e. those passed to `logger.Noticef`) should begin with a capital letter, and use "Cannot X" rather than "Error Xing":
```

Comment:
```suggestion
**Log messages** (that is, those passed to `logger.Noticef` or `logger.Debugf`) should begin with a capital letter, and use "Cannot X" rather than "Error Xing":
```